modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id=using-cluster-logging-to-find-Knative-Serving-logs_{context}"]
+= Using Cluster Logging to find logs for Knative Serving components
+
+.Prerequisites
+
+* xref:../efk-logging-about_{context}[Cluster Logging components] are installed.
+
+.Procedure
+
+. To open the Kibana UI (the visualization tool for Elasticsearch), use the following command to get the Kibana route:
+
+`oc -n openshift-logging get route kibana`
```

Comment:
The command should go in a code block preceded by a "+" (the "+" will keep the code block in line with the numbered list). 
+
`-` x 4 (4 hyphens `----` - Github does some wonky formatting)
$ oc -n openshift-logging get route kibana
`-` x 4